website/home/app/page.tsx

```diff
@@ -153,29 +163,28 @@ const agent = new Agent({
   }[] = [
     {
       type: "components",
-      title: "Components",
-      mobileTitle: "Components",
+      title: "Create Components",
+      mobileTitle: "Create Components",
       description:
         "Create building blocks for your app with reusable components.",
     },
     {
       type: "workflows",
-      title: "Workflows",
-      mobileTitle: "Workflows",
-      description: "Use Components to build and run workflows.",
+      title: "Run Workflows",
+      mobileTitle: "Run Workflows",
+      description: "Combine components to build and run powerful workflows.",
     },
     {
       type: "agents",
-      title: "Agentic Patterns",
-      mobileTitle: "Agentic Patterns",
-      description: "Expressive and powerful agentic patterns.",
+      title: "Build Agents",
+      mobileTitle: "Build Agents",
+      description: "Create agents to handle complex tasks.",
     },
     {
       type: "llms",
-      title: "Composable and Reusable",
-      mobileTitle: "LLMs",
-      description:
-        "Install or publish components on npm and use throughout your workflows.",
+      title: "Use any LLM",
+      mobileTitle: "Use any LLM",
+      description: "Easily swap between models and providers.",
```

Comment:
Another option for this one could be one around showing how to use components from a fictional package to demonstrate that you can use components from npm and publish them there

Reply:
or could be a 5th example if we wanted